speech/speech_samples.rb

```diff
@@ -12,46 +12,89 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Required to load locally generated Cloud Speech API client
-# TODO remove once Cloud Speech API client is released in google-api-client gem
-$LOAD_PATH.unshift File.expand_path("generated", __dir__)
+def speech_sync_recognize project_id:, audio_file_path:
+# [START speech_sync_recognize]
+  # project_id      = "Your Google Cloud project ID"
+  # audio_file_path = "Path to file on which to perform speech recognition"
 
-def initialize_speech_client
-  # [START initialize_speech_client]
-  require "google/apis/speech_v1beta1"
+  require "google/cloud/speech"
 
-  speech_service = Google::Apis::SpeechV1beta1::SpeechService.new
+  speech = Google::Cloud::Speech.new project_id: project_id
```

Comment:
Classes use named parameter `project` instead of `project_id`. Review Circle CI logs for errors.